app.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const app = express();
+
+
+const magic8Responses = [
+    "It is certain",
+    "It is decidedly so",
+    "Without a doubt",
+    "Yes - Definitely",
+    "You may rely on it",
+    "As I see it, yes",
+    "Most likely",
+    "Outlook good",
+    "Yes, and signs point to yes",
+    "Reply hazy, try again",
+    "Ask again later",
+    "Better not tell you now",
+    "Cannot predict now",
+    "Concentrate and ask again",
+    "Don't count on it",
+    "My reply is no",
+    "My sources say no",
+    "Outlook not so good",
+    "Very doubtful"
+  ]
+
+  const random = Math.floor(Math.random()*magic8Responses.length);
```

Comment:
Because you define this variable in the global scope, it doesn't change its value after the first time the file runs. Move this logic into the correct route so it will be recreated every time you hit the route for `/magic8`